openapi/nexus.json

```diff
@@ -6226,6 +6332,194 @@
       }
     },
     "schemas": {
+      "BinRangedouble": {
```

Comment:
This comes out perfectly reasonable from the TS generator. Where does the janky name come from though?

```ts
export type BinRangedouble =
  | { end: number; type: 'range_to' }
  | { end: number; start: number; type: 'range' }
  | { start: number; type: 'range_from' }
```

Reply:
It's a generic: `BinRange<T>`. `BinRange<f64>` -> `BinRangedouble` apparently.